### PR TITLE
doc(README) : Adding how to debug the project

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
Adding how to debug the project to the README

## Progress

- [x] Finished task
- [ ] TODO

## Additional Notes

Screenshot 2019-04-28 at 20 21 28

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -86,7 +86,7 @@ $ make create-all
 
 IMPORTANT: To create all you need be logged as admin. Use `oc login -u system:admin`.
```
Comment (camilamacedo86) [inline README.adoc]: Small fix typo.

File: README.adoc [hunk 2]
```diff
@@ -266,6 +266,58 @@ $ make run-local
 
 IMPORTANT: The local changes are applied when the command `operator-sdk up local --namespace=mobile-security-service-operator` is executed then it is not a hot deploy and to get the latest changes you need re-run the command.
 
+
+=== Debugging
+
+Following the steps to debug the project in some IDEs.
+
+NOTE: The code need to be compiled/built first.
```
Comment (davidffrench) [inline README.adoc]:
```suggestion
NOTE: The code needs to be compiled/built first.
```

File: README.adoc [hunk 3]
```diff
@@ -266,6 +266,58 @@ $ make run-local
 
 IMPORTANT: The local changes are applied when the command `operator-sdk up local --namespace=mobile-security-service-operator` is executed then it is not a hot deploy and to get the latest changes you need re-run the command.
 
+
+=== Debugging
+
+Following the steps to debug the project in some IDEs.
```
Comment (davidffrench) [inline README.adoc]:
```suggestion
Follow the below steps to debug the project in some IDEs.
```

File: README.adoc [hunk 4]
```diff
@@ -266,6 +266,58 @@ $ make run-local
...
+
+==== Visual Code
+
+[source,shell]
+----
+$ export WATCH_NAMESPACE=default
+$ $ dlv --listen=:2345 --headless=true --api-version=2 exec ./build/_output/bin/mobile-security-service-operator-local  --
+----
+
+Then, attach to process using the following Visual Code config.
```
Comment (davidffrench) [inline README.adoc]:
```suggestion
Then, debug the project using the following Visual Code launch config.
```

File: README.adoc (hunk 3 above)
Comment (camilamacedo86) [reply README.adoc]: 👍

File: README.adoc (hunk 2 above)
Comment (camilamacedo86) [reply README.adoc]: 👍

File: README.adoc (hunk 4 above)
Comment (camilamacedo86) [reply README.adoc]: 👍

**Pull request merged**